systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceManager.java

```diff
@@ -426,14 +426,14 @@ public static String getImageValueFromCO(String name) {
      * @param status - desired status
      * @return returns CR
      */
-    public static <T extends HasMetadata & HasStatus> T waitForResourceStatus(MixedOperation<T, ?, ?, ?> operation, T resource, String status, long resourceTimeout) {
+    public static <T extends HasMetadata & HasStatus> T waitForResourceStatus(MixedOperation<T, ?, ?, ?> operation, T resource, Object status, long resourceTimeout) {
```

Comment:
Here you can use an enum type `CustomResourceStatus` instead of comprehensive `Object`

Reply:
Well not in this case, because I'm also using the enum `KafkaRebalanceState` for `KafkaRebalance` as you can see here: https://github.com/strimzi/strimzi-kafka-operator/pull/3452/files/06a76c49b77412777e45e22fcd6778871b4dc40e#diff-a308e0a56c31c930d11b603227e21b2bR81